kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/dissolution_chamber.js

```diff
@@ -0,0 +1,71 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+
+    const recipes = [
+/*
+        {
+            inputs: [
+                '', // top left
+                '', // top 
+                '', // top right
+                '', // left
+                '', // right
+                '', // bottom left
+                '', // bottom
+                ''  // bottom right
+            ],
+            inputFluid: '', // optional
+            inputFluidAmount: 0, // leave at 0 for no fluid
+            processingTime: 50,
+            outputItemName: '',
+            outputItemCount: 1,
+            outputFluid: '', // optional
+            outputFluidAmount: 0, // leave at 0 for no fluid
+            id: ''
+        }
+*/
+        {
+            inputs: [
+                'minecraft:redstone',
+                'rftoolsbase:machine_frame',
+                'minecraft:redstone',
+                'minecraft:redstone_torch',
+                'minecraft:redstone_torch',
+                '#forge:ingots/gold_copper',
+                '#forge:ingots/gold_copper',
+                '#forge:ingots/gold_copper'
+            ],
+            inputFluid: 'tconstruct:molten_obsidian',
+            inputFluidAmount: 1000,
+            processingTime: 600,
+            outputItem: 'rftoolsbuilder:shield_block1',
+            outputItemCount: 1,
```

Comment:
I recommend merging outputItem & outputItemAmount into outputItem. It would like like this:

```js
output: { item: 'rftoolsbuilder:shield_block1', count: 1 }
```

and usage
```js
output: recipe.output,
```

Reply:
Can do; was trying to make it "simpler" but possibly went too far ^_^

Reply:
Can also do the same for input items where appropriate.

Reply:
It is a bit simpler, but it doesn't support tags which is unfortunate.

I think it works fine for input fluids as you made it though :)